### DISCUSSION TITLE
Homebrew fonts/taps support

### DISCUSSION THREAD

**Discussion by tnxz** [milanvarady/Applite#62] [Q&A] [answered]
are homebrew fonts supported and in general homebrew taps ?

**Comment by milanvarady** [comment, marked as answer]
Support for homebrew taps may be added in the future. If this happens any font that is a `cask` should be available through the search feature in Applite.

**Reply by milanvarady** [reply]
Tap support is now available in the [v1.3.0 update](https://github.com/milanvarady/Applite/releases/tag/v1.3.0).
